src/cljs/webchange/editor_v2/voice_over_display/views.cljs

```diff
@@ -0,0 +1,53 @@
+(ns webchange.editor-v2.voice-over-display.views
```

Comment:
Move this component to `webchange/editor_v2/dialog/dialog_text_form/side_menu/main/voice_over/` because currently this modal window works only with `dialog-text-form` state